arangod/Aql/QuerySnippet.h

```diff
@@ -81,7 +81,7 @@ class QuerySnippet {
   Id id() const { return _id; }
 
  private:
-  ResultT<std::unordered_map<ExecutionNode*, std::set<ShardID>>> prepareFirstBranch(
+  ResultT<std::unordered_map<ExecutionNode*, std::unordered_map<std::string, std::set<ShardID>>>> prepareFirstBranch(
```

Comment:
why don't you use a typedef for this stuff?

Reply:
Yes good point, we can use a typedef here.